lms/djangoapps/grades/tasks.py

```diff
@@ -54,8 +55,26 @@ class _BaseTask(PersistOnFailureTask, LoggedTask):  # pylint: disable=abstract-m
     abstract = True
 
 
+@task(base=_BaseTask)
+def compute_all_grades_for_course(**kwargs):
```

Comment:
Can we refactor this so we reuse the same code that already exists in `grades/management/commands/course_grades.py`?